client_code/kompot/_batcher.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+
+import anvil.server
+from anvil import is_server_side
```

Comment:
oohh! Hello!

Reply:
Welcome to the party. I don't believe we've met before.

Reply:
super useful!

Reply:
Have I just missed it somewhere? I can't see it listed at https://anvil.works/docs/api/anvil

Reply:
probably not - it comes up in my editor's LSP.
I only know about it because of recent work on some anvil internal server stuff.

i'll get it added to the auto completions/api docs.